app/views/all_casa_admins/casa_orgs/show.html.erb

```diff
@@ -30,3 +30,30 @@
     </table>
   </div>
 </div>
+
+<div class="card card-container">
+  <div class="card-body">
+    <div class="card-title">
+      <h3>Details</h3>
+    </div>
+    <ul class="list-group list-group-flush">
+      <%{
+        "Number of admins" => selected_organization.casa_admins.count,
+        "Number of supervisors" => selected_organization.supervisors.count,
+        "Number of active volunteers" => selected_organization
+          .volunteers.where(active: true).count,
+        "Number of inactive volunteers" => selected_organization
+          .volunteers.where(active: false).count,
+        "Number of active cases" => selected_organization
+          .cases.where(active: true).count,
+        "Number of inactive cases" => selected_organization
+          .cases.where(active: false).count,
+      }.each do |item_description, item_value|%>
+        <li class="list-group-item">
+          <strong><%=item_description%>:</strong>
+          <%=item_value%>
+        </li>
+      <% end %>
+    </ul>
+  </div>
+</div>
```

Comment:
This missing EOL shows up in github and it should upset the linter but it does not?? We should fix that